docs/explanation/parts.rst

```diff
@@ -0,0 +1,263 @@
+Parts
+=====
+
+In the Craft Parts framework, parts are descriptions of the components to be
+build for deployment, either individually or as part of a larger build
+containing many components.
+
+Tools like `Snapcraft`_ and `Juju`_ that use the concepts of parts to
+describe a build process typically accept specifications of parts in YAML format. This allows each part to be described in a convenient,
```

Comment:
typically accept - is there a different way?

Reply:
I'm thinking about tools that use the `craft_parts` library directly. I guess they might use the Python data structures directly?

Reply:
That's correct, Craft Parts receives parts data as a Python dictionary.